cmd/integration-test-server/main.go

```diff
@@ -175,7 +178,14 @@ func checkFinalState(state *node.NodeState) {
 			addErrorMessage("At lease one block should be mined successfully during the test.")
 		}
 		if shardState.SubmissionState.Failed > 0 {
-			addErrorMessage(fmt.Sprintf("%d submission failed during the test.", shardState.SubmissionState.Failed))
+			byDesignCount := checkByDesignFailure()
```

Comment:
I think "by design" is very misleading. They are undesired edge cases that may only occur upon intensive mining TX submissions like integration tests.